backupctl.8.rst

```diff
@@ -81,6 +82,30 @@ OPTIONS
                         human readable as MB, GB and so on.
 
 
+QUOTA
+======
+The data in a dirvish vault is used from the quota of the vault/server and of
+the customer. The quota of the customer should be equal or bigger as the one
+of the vault/server, if this isn't the case, then the quota of the customer
+can't be used in the full size. A quota is needed for each customer, but not
```

Comment:
Rephrase: A quota must be defined for a customer. It may be defined for a vault/server.
https://tools.ietf.org/html/rfc2119